src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -1218,13 +1216,7 @@ uint64_t GCToOSInterface::GetPhysicalMemoryLimit(bool* is_restricted)
     if (is_restricted)
         *is_restricted = false;
 
-    // The limit was not cached
```

Comment:
GetPhysicalMemoryLimit memory is called from multiple places. Is it intentional to disable the cache for all callers and not just `refresh_memory_limit`?